spec-main/api-screen-spec.ts

```diff
@@ -1,7 +1,19 @@
 import { expect } from 'chai';
 import { screen } from 'electron/main';
+import * as sinon from 'sinon';
 
 describe('screen module', () => {
+  describe('methods reassignment', () => {
+    after(() => {
+      sinon.restore();
+    });
+
+    it('works for a selected method', () => {
+      sinon.stub((screen as any), 'getPrimaryDisplay').returns(null);
+      expect(screen.getPrimaryDisplay()).to.be.null();
+    });
+  });
```

Comment:
i'd prefer we test this more directly, without involving a 3rd-party dependency. perhaps:
```suggestion

describe('screen module', () => {
  describe('methods reassignment', () => {
    it('works for a selected method', () => {
      const originalFn = screen.getPrimaryDisplay
      try {
        (screen as any).getPrimaryDisplay = () => null;
        expect(screen.getPrimaryDisplay()).to.be.null();
      } finally {
        (screen as any).getPrimaryDisplay = originalFn
      }
    });
  });
```

Reply:
@nornagon, why not, done.